Sources/Identity/ProductPaidPrice.swift

```diff
@@ -22,12 +22,46 @@ import Foundation
     /// Amount paid
     @objc public let amount: Double
 
+    /// Formatted price of the item, including its currency sign. For example $3.00.
+    @objc public let formatted: String
+
+    private static let numberFormatter: NumberFormatter = {
+        let formatter = NumberFormatter()
+        formatter.numberStyle = .currency
+        return formatter
+    }()
+
     /// ProductPaidPrice initialiser
     /// - Parameters:
     ///   - currency: Currency paid
     ///   - amount: Amount paid
-    public init(currency: String, amount: Double) {
```

Comment:
Hmm interesting, I guess this is not breaking in ObjC because it was never exposed in ObjC 😅. I think it's fine keeping it like this though, we can expose it later if needed.